pkg/cloudprovider/fake/instancetype.go

```diff
@@ -65,6 +68,40 @@ func NewInstanceType(options InstanceTypeOptions) *InstanceType {
 	}
 }
 
+// InstanceTypesAssorted create 1000+ unique instance types with varying CPU/memory/architecture/OS/zone/capacity type.
+func InstanceTypesAssorted() []cloudprovider.InstanceType {
+	var instanceTypes []cloudprovider.InstanceType
+	for _, cpu := range []int{1, 2, 4, 8, 16, 32, 64} {
+		for _, mem := range []int{1, 2, 4, 8, 16, 32, 64, 128} {
+			for _, zone := range []string{"test-zone-1", "test-zone-2"} {
```

Comment:
would adding a 3rd zone break anything? 3 zones is a more typical "real world" setup

Reply:
If it does, our code is broken, adding a third zone.